src/axolotl/cli/config.py

```diff
@@ -233,4 +234,10 @@ def load_cfg(
     setup_comet_env_vars(cfg)
     plugin_set_cfg(cfg)
 
+    redacted_cfg = redact_sensitive_info(cfg)
+    LOG.info(
+        "config:\n%s",
+        json.dumps(redacted_cfg, indent=2, default=str, sort_keys=True),
```

Comment:
Does this need to `sort_key=True`? I think it'll make it hard to cross check with their config easily.

Reply:
Personally I found it easier to find what I was looking for 🤔 but you raise a good point. Could go either way IMO.